README.md

```diff
@@ -48,7 +50,7 @@ If you have Jupyter installed, you can create a notebook using IRkernel from the
 You can also start other interfaces with an R kernel:
 
 ```bash
-# “ir” is the kernel name installed by the above 'IRkernel::installspec()'
```

Comment:
I think that's probably fair enough with code samples. This is a bit of a corner case because it's a comment, but someone may still copy and paste the whole block, and I don't think smart quotes are that important. If it was someone's name with a non-ascii character in, I'd take it more seriously, but let's not waste time worrying about quotes.